BPL-CBH LCCN

```diff
@@ -0,0 +1,89 @@
+'MacroName:BPLLCCN
+'MacroDescription:LC Call version for BPL
+   
+   'LC Call number
+'Created by Miriam Gloger 
+'Copy Call number from 050 to 099 Call number for Brooklyn Public Library CBH
+
+Option Explicit
+
+Sub Main
+   Dim CS as Object
+   On Error Resume Next
+   Set CS  = GetObject(,"Connex.Client")
+   On Error GoTo 0
+   If CS  Is Nothing Then
+      Set CS  = CreateObject("Connex.Client")
+   End If
+  
```

Comment:
Maybe it is a good idea to add safe guards here so the macro cannot be executed on a different window than one displaying bibliographic record. Check OCLC documentation for correct `CS.ItemType`.